### DISCUSSION TITLE
Warning! Some values within column "id" of the rowData (gene metadata) of your objects do not precisely match!

### DISCUSSION THREAD

**Discussion by sylestiel** [GreenleafLab/ArchR#1506] [General]
How can I address the mismatch error shown below when I run import10xFeatureMatrix()for Multiome datasets. Not sure if it is a bug. The log file just shows the creation of arrow files or the previous step log. 
[ArchR-createArrows-b08021299c32-Date-2022-07-13_Time-17-13-28.log](https://github.com/GreenleafLab/ArchR/files/9106460/ArchR-createArrows-b08021299c32-Date-2022-07-13_Time-17-13-28.log)

> seRNA <- import10xFeatureMatrix(
+     input = c("/Volumes/BKUP2/E14.5_IUGR_Multiome_data/scMultiomeATAC_GEX/scMultiome_5/outs/filtered_feature_bc_matrix.h5","/Volumes/BKUP2/E14.5_IUGR_Multiome_data/scMultiomeATAC_GEX/scMultiome_2/Outs/filtered_feature_bc_matrix.h5"),
+     names = c("NPD3","LPD2")
+ )

### Importing Feature Matrix 1 of 2
Importing Feature Matrix 2 of 2
Re-ordering RNA matricies for consistency.
Merging individual RNA objects...

Merging LPD2

Warning! Some values within column "id" of the rowData (gene metadata) of your objects do not precisely match!
This is often caused by slight variations in Ensembl IDs and gene locations used by cellranger across different samples. ArchR will ignore these mismatches and allow merging to proceed but you should check to make sure that these are ok for your data.

Mismatch in column "id" row 17711 for LPD2: ENSMUSG00000112189 does not exactly match ENSMUSG00000112366!
strictMatch = TRUE so the corresponding gene entry with mismatching information will be removed.
Mismatch in column "id" row 26278 for LPD2: ENSMUSG00000116197 does not exactly match ENSMUSG00000115968!
strictMatch = TRUE so the corresponding gene entry with mismatching information will be removed.

**Comment by rcorces** [comment]
This is certainly not a bug. I dont have much to add beyond the very detailed warning messages that I coded for this.

**Reply by sylestiel** [reply]
So no way to address this problem?

**Reply by rcorces** [reply]
Please read the updated function parameter definitions, in particular `strictMatch`

**Reply by sylestiel** [reply]
I did a search and could not locate it. Care to provide a link?

**Reply by rcorces** [reply]
The documentation on the website is in the process of being updated. The most effective way of finding documentation for the version of ArchR you are using is to type"?" followed by the function of interest.
`?import10xFeatureMatrix`
